tests/cmake_integration/Messenger/Messenger_2/CMakeLists.txt

```diff
@@ -38,19 +39,17 @@ foreach(t ${all_targets})
 endforeach()
 
 # Copy configs/scripts into build-output directory
-if (EXISTS ${DDS_ROOT}/tests/DCPS/Messenger)
```

Comment:
This if-block exists to enable these tests to work outside of the source tree (e.g. with a make-installed OpenDDS)—these should not be removed.